### PR TITLE
Develop/turbo custom with UI without tpv

### PR DESCRIPTION
**Pull request by AtifQamarRaz**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java [hunk 1]
```diff
@@ -107,28 +119,114 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
...
+        break;
+
+      case "manageUpiAccounts":
+        customerMobile = call.arguments.toString();
+        razorpayDelegate.manageUpiAccounts(customerMobile, result , this.eventSink);
+        break;
+
       default:
-        Log.d("RAZORPAY_SDK","no method");
+        Log.d(TAG,"no method");
```
Comment (shibu000) [inline android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java:196]: @AtifQamarRaz Please give a proper Log message here.

File: android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java [hunk 2]
```diff
@@ -107,28 +119,114 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
...
   @Override
   public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
     channel.setMethodCallHandler(null);
+    eventChannel.setStreamHandler(null);
+    this.eventChannel.setStreamHandler(null);
   }
 
   @RequiresApi(api = Build.VERSION_CODES.KITKAT)
   public RazorpayPlugin(Registrar registrar) {
+    this.activity = registrar.activity();
```
Comment (shibu000) [inline android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java:209]: @AtifQamarRaz Typo?

File: example/lib/main.dart [hunk 3]
```diff
@@ -19,6 +19,7 @@ class _MyAppState extends State<MyApp> {
         primarySwatch: Colors.blue,
       ),
       home: MyHomePage(),
+
```
Comment (shibu000) [inline example/lib/main.dart:22]: @AtifQamarRaz Remove the extra space.

**Review by shibu000 — COMMENTED** [review]
Please check the comments.

File: android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java (hunk 1 above)
Comment (AtifQamarRaz) [reply android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java:196]: @shibu000 it was already there. Should I change this?

**Review by shibu000 — APPROVED** [review]
Looks good to me

**Pull request merged**